src/views/admin/PendingAccounts.vue

```diff
@@ -3,17 +3,17 @@
     <div class="container mt-5">
       <div class="columns is-tablet">
         <div class="column">
-           <loading v-model:active="isLoading"
-                 :can-cancel="false"
-                 :is-full-page= "true"/>
-          <div
-            v-if="!isLoading" class="accounts card custom-card custom-card-padding"
-          >
-         <div v-if="pendingUserAccounts.length === 0">Aucun compte en attente de validation</div>
+          <div class="accounts card custom-card custom-card-padding">
          
-        <div v-else>
-         <h2 class="custom-card-title">Comptes en attente de validation</h2>
-          <table   class="table is-striped is-fullwidth">
+          <h2 class="custom-card-title">Comptes en attente de validation</h2>
```

Comment:
To me its kinda awkward to either have a title, or not depending of the contents displayed. There should be one, always, or never.

It's the same thing as having a loading-indicator make a title hide, and then displayed. The user should know about what is actually loading underneath.

Reply:
All this is important since it's part of the user experience. But it is subjective and YMMV.
In the end I don't have a strong opinion. Just did as I felt at the moment.

Changed.

Reply:
You have your point about having a title while loading. My point was only that the title is a little awkward once we know the list is empty (after the loading). No strong feelings about this, and I don't think it is this important either way.

Reply:
Me neigher so I did as you suggested.